src/modules/User/model.ts

```diff
@@ -44,11 +44,11 @@ export function getUserCrud(prisma: PrismaClient) {
         },
       });
       if (!user) {
-        throw new ApolloError('User not found', 'USER_NOT_FOUND');
+        throw new GraphQLError('User not found', { extensions: { code: 'USER_NOT_FOUND' } });
```

Comment:
Curious, what's the difference between `ApolloError` and `GraphQLError` and why did you switch?

Reply:
Since i've updated the system from the deprecated https://www.npmjs.com/package/apollo-server-express to latest https://www.apollographql.com/docs/apollo-server/ , the error handling is now done according to the corresponding docs https://www.apollographql.com/docs/apollo-server/data/errors/#custom-errors